spec/hosts/nodes/auxtel-archiver.cp.lsst.org_spec.rb

```diff
@@ -29,72 +29,6 @@
 
       it { is_expected.to compile.with_all_deps }
 
-      include_examples 'baremetal'
```

Comment:
Could you include the `vm` examples?

Reply:
had to open the shared examples because the active_profile on the VMs in the center is set to "balanced"

Reply:
That is a bug we should fix. https://github.com/lsst-it/lsst-control/blob/production/hieradata/common/virtual/kvm.yaml needs to be copied to be `vmware.yaml` and that should set the expected tuned profile.